src/modules/file-share/file-share-list/file-share-list.scss

```diff
@@ -4,14 +4,6 @@
         font-weight: bold;
     }
 
-    .counts {
-        color: $color-boston-blue;
-
-        &:hover {
-            cursor: pointer;
-        }
-    }
-
```

Comment:
@gnehapk I think we don't need it from now. This is because we added blue color for showing the link. Which isn't a good way.